### PR TITLE
Telia Parser

### PR DESCRIPTION
**Pull request by carbonarok**
Enable Telia parser for circuit maintenance plugin. Using ICal parsing Telia emails are very easy thanks to Christian.

### CODE REVIEW

File: circuit_maintenance_parser/providers.py [hunk 1]
```diff
@@ -125,6 +125,13 @@ class PacketFabric(GenericProvider):
     _default_organizer = "support@packetfabric.com"
 
 
+class Telia(GenericProvider):
+    """Telia provider custom class."""
+
+    _parser_classes: Iterable[Type[Parser]] = [ICal]
+    _default_organizer = "ncm@teliacompany.com"
```
Comment (glennmatthews) [inline circuit_maintenance_parser/providers.py:132]: Is there a reason the default organizer here doesn't match the ORGANIZER in the unit test?

**Review by glennmatthews — COMMENTED** [review]
Nice! I hadn't realized Telia was using the standard - that's great news!

Probably should add this to the list of supported providers in the README as well.

File: circuit_maintenance_parser/providers.py (hunk 1 above)
Comment (chadell) [reply circuit_maintenance_parser/providers.py:132]: I believe it's because if the notification contains a organizer, the default is not used

**Comment by chadell** [comment]
@carbonarok I believe that this is a great opportunity to add the dual maintenance in a notification test, would you do it?

File: tests/unit/test_providers.py [hunk 2]
```diff
@@ -94,14 +109,14 @@ def test_complete_provider_process(provider_class, raw_file, results_file):
     # If the Provider test is using the GENERIC_ICAL_DATA_PATH it comes with a well-defined 'organizer'
     # from the notificaction.
     if provider_class == GenericProvider or raw_file == GENERIC_ICAL_DATA_PATH:
-        expected_result["organizer"] = "mailto:noone@example.com"
+        expected_result[0]["organizer"] = "mailto:noone@example.com"
```
Comment (chadell) [inline tests/unit/test_providers.py:112]: @carbonarok why not just put this check into a `for loop`?

Comment (carbonarok) [reply tests/unit/test_providers.py:112]: ah yeah good idea

**Review by chadell — DISMISSED** [review]
LGTM, only point would be to add the new feature to Changelog

**Review by chadell — APPROVED** [review]
LGTM 🏅

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**